test/Table_Tests/src/In_Memory/Split_Tokenize_Spec.enso

```diff
@@ -0,0 +1,171 @@
+
+        Test.specify 'can do tokenize_to_rows case-insensitively' <|
+            cols = [['foo', [0, 1, 2]], ['bar', ['aBqcE', 'qcBr', 'cCb']]]
+            t = Table.new cols
+            expected_rows = [[0, 'B'], [0, 'c'], [1, 'c'], [1, 'B'], [2, 'c'], [2, 'C'], [2, 'b']]
+            expected = Table.from_rows ['foo', 'bar'] expected_rows
+            t2 = t.tokenize_to_rows 'bar' "[bc]" case_sensitivity=Case_Sensitivity.Insensitive
+            tables_should_be_equal t2 expected
+
+    Test.group 'column count' <|
+        Test.specify 'should generate extra empty columns if column_count is set' <|
+            cols = [['foo', [0, 1, 2]], ['bar', ['abc', 'cbdbef', 'ghbijbu']]]
+            t = Table.new cols
+            expected_rows = [[0, 'a', 'c', Nothing, Nothing], [1, 'c', 'd', 'ef', Nothing], [2, 'gh', 'ij', 'u', Nothing]]
+            expected = Table.from_rows ['foo', 'bar_0', 'bar_1', 'bar_2', 'bar_3'] expected_rows
+            t2 = t.split_to_columns 'bar' 'b' column_count=4
+            tables_should_be_equal t2 expected
+
+        Test.specify "split should limit columns and return problems when exceeding the column limit" <|
+            cols = [['foo', [0, 1, 2]], ['bar', ['abc', 'cbdbef', 'ghbijbu']]]
+            t = Table.new cols
+            expected_rows = [[0, 'a', 'c'], [1, 'c', 'd'], [2, 'gh', 'ij']]
+            expected = Table.from_rows ['foo', 'bar_0', 'bar_1'] expected_rows
+            action = t.split_to_columns 'bar' 'b' column_count=2 on_problems=_
+            tester = t-> tables_should_be_equal t expected
+            problems = [Column_Count_Exceeded.Error 2 3]
+            Problems.test_problem_handling action problems tester
+
+        Test.specify "tokenize should limit columns and return problems when exceeding the column limit" <|
+            cols = [['foo', [0, 1]], ['bar', ['r a-1, b-12,qd-50', 'ab-10:bc-20c']]]
+            t = Table.new cols
+            expected_rows = [[0, 'a1', 'b12', 'd50'], [1, 'b10', 'c20', Nothing]]
+            expected = Table.from_rows ['foo', 'bar_0', 'bar_1'] expected_rows
+            action = t.tokenize_to_columns 'bar' "([a-z]).(\d+)" column_count=2 on_problems=_
+            tester = t-> tables_should_be_equal t expected
+            problems = [Column_Count_Exceeded.Error 2 3]
+            Problems.test_problem_handling action problems tester
+
+        Test.specify 'should generate extra empty columns if column_count is set (reverse row order)' <|
+            cols = [['foo', [0, 1, 2]], ['bar', ['ghbijbu', 'cbdbef', 'abc']]]
+            t = Table.new cols
+            expected_rows = [[0, 'gh', 'ij', 'u', Nothing], [1, 'c', 'd', 'ef', Nothing], [2, 'a', 'c', Nothing, Nothing]]
+            expected = Table.from_rows ['foo', 'bar_0', 'bar_1', 'bar_2', 'bar_3'] expected_rows
+            t2 = t.split_to_columns 'bar' 'b' column_count=4
+            tables_should_be_equal t2 expected
+
+    Test.group 'errors' <|
```

Comment:
minor nitpick: I'd name this group `Split/Tokenize errors`.

Our test suite only prints group and spec names and does not print the filename (actually I think it's something we maybe should change?). So without this a group just called `errors` is _very_ generic.